packages/product-react/src/components/Select/Select.tsx

```diff
@@ -8,9 +8,9 @@ import RTLWrapper from '../Common/RTLWrapper';
 import './Select.scss';
 
 export enum IconPosition {
-  Prepend = 'PREPEND',
-  End = 'END',
-  Start = 'START'
+  PREPEND = 'PREPEND',
```

Comment:
What's the purpose of this change?

Reply:
This to add the prepend icon for the select component, just in case the user wants it
![Screen Shot 2019-11-01 at 1 38 00 PM](https://user-images.githubusercontent.com/45638740/68044153-26ae2700-fcad-11e9-9e6d-fdd463bfce2b.png)